### ISSUE TITLE
JavaScript not working. 

### ISSUE THREAD

**Issue by Bunnybailey** [TheLastBen/fast-stable-diffusion#2376]
I have been trying to use the notebook now for the last couple of days, but sadly it keeps giving me the same error. does anyone know what would be the cause? Below is the error message i keep getting and I have signed back in, I've signed into the incognito mode, I have tried to change my settings on google to fix it, but no luck.

Error
Could not load the JavaScript files needed to display output. This is probably because your Google Account login access has expired or because third-party cookies are not allowed by your browser. Please reload this page.

**Comment by TheLastBen** [comment]
Change a browser

**Comment by Bunnybailey** [comment]
yep! That seemed to have worked. Thank you! Weird that it suddenly stopped working.

**Comment by TheLastBen** [comment]
great!
